src/modules/httptrap.c

```diff
@@ -457,6 +458,12 @@ rest_get_json_upload(mtev_http_rest_closure_t *restc,
 
   content_length = mtev_http_request_content_length(req);
   rxc = restc->call_closure;
+  rxc->check = noit_poller_lookup(rxc->check_id);
+  if (!rxc->check) {
+    *complete = 1;
```

Comment:
This isn't necessarily an error.... it likely means that a check was deleted while in the middle of processing and that the rest of the data should be discarded. I think we could potentially spam too much stuff in the error log here if people constantly create and delete a bunch of httptrap checks.